libs/SmartSync/SmartSync/Classes/Util/SFSyncOptions.h

```diff
@@ -40,6 +44,11 @@ extern NSString * const kSFSyncOptionsMergeMode;
 + (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist;
 + (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist mergeMode:(SFSyncStateMergeMode)mergeMode;
 
++ (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist
```

Comment:
I understand the "base method" idea being employed here, but does it actually make sense in the public interface?  It seems like the inclusion of all three of these—or really any configuration that includes both `fieldList` and one of the other new collections—makes for ambiguous intent.  For example, if I have all three, what does that mean?